parse_blocks.pl

```diff
@@ -27,30 +27,57 @@
 # TODO FIX THIS ^
 
 #returns list of string blocks from stdin
+
+
+sub does_line_match_end_block {
+  my $line = shift;
+
+  # Make sure you take into account random whitespace that could happen by using [ ]*[\t]*
+  my $match = 0;
+  $match |= $line =~ m/^};[ ]*[\t]*\/\//;  # Semicolon,    comment
+  $match |= $line =~ m/^};[ ]*[\t]*$/;     # Semicolon,    no comment
+  $match |= $line =~ m/^}[ ]*[\t]*\/\//;   # No semicolon, comment
+  $match |= $line =~ m/^}[ ]*[\t]*$/;      # No semicolon, no comment
```

Comment:
We might be able to change all this just by searching for /^}/ and being agnostic to what comes afterwards. It works properly like that in v1.60 at least from testing.